### PR TITLE
Add those single letter directives to index.

### PR DESCRIPTION
**Pull request by thdox**

### CODE REVIEW

File: doc/ledger3.texi [hunk 1]
```diff
@@ -2554,12 +2555,15 @@ alone, for backwards compatibility with older Ledger versions.
 @table @code
 
 @item A
+@findex A
 See @code{bucket}.
```
Comment (afh) [inline doc/ledger3.texi]: Is it possible to use a `@pxref` to bucket here?

File: doc/ledger3.texi [hunk 2]
```diff
@@ -2554,12 +2555,15 @@ alone, for backwards compatibility with older Ledger versions.
...
 
 @item Y
+@findex Y
 See @code{year}.
```
Comment (afh) [inline doc/ledger3.texi]: Is it possible to use a `@pxref` to year here?

File: doc/ledger3.texi (hunk 1 above)
Comment (thdox) [inline doc/ledger3.texi]: Good, had to learn a new thing !  :-) I pushed a new pull request.

Comment (afh) [inline doc/ledger3.texi]: Merged it. Thanks! :)

**Pull request closed without merging**